apps/rewards/app/components/Content/MyRewards.js

```diff
@@ -1,8 +1,13 @@
 import PropTypes from 'prop-types'
 import React from 'react'
 import styled from 'styled-components'
+import moment from 'moment'
```

Comment:
I see you've already started using `moment` throughout Rewards, but we do not have `moment` in our `package.json` dependencies, and in all other places where we've been doing date calculations, we've been using `date-fns`. Personally, I love `date-fns`, while I've had very frustrating experiences with `moment`, so I'm really glad we don't use `moment`! However, if you think there's a case to be made that we switch to `moment` instead, I think we should have that discussion in an isolated PR later. For now, I think this PR needs to avoid using `moment`, and I think we need a separate PR that removes `moment` from the places where you've added it already.